README.md

```diff
@@ -29,6 +29,7 @@ limitations under the License.
     - [LLama 70b BF16 on 8 Gaudi2 card](#llama-70b-bf16-on-8-gaudi2-card)
     - [LLama 70b FP8 on 8 Gaudi2 card](#llama-70b-fp8-on-8-gaudi2-card)
     - [LLava-next 7B BF16 on 1 Gaudi2 card](#llava-next-7b-bf16-on-1-gaudi2-card)
+    - [LLava-next 7B FP8 on 1 Gaudi2 card](#llava-next-7b-fp8-on-1-gaudi2-card)
```

Comment:
@yuanwu2017 , there are quite a few variants for Llava-next. Please list those which were supported here, such as this README: https://github.com/huggingface/optimum-habana/tree/main/examples/image-to-text